blocks/library/quote/style.scss

```diff
@@ -0,0 +1,24 @@
+.blocks-quote {
+	margin: 0;
+	box-shadow: inset 0px 0px 0px 0px $light-gray-500;
+	transition: all .2s ease;
+	font-size: 20px;
+	border-left: 4px solid $black;
+	padding-left: 1em;
+	font-style: italic;
+
+	footer {
+		color: $dark-gray-100;
+		font-size: 0.9em;
+		font-style: normal;
+		margin-left: 1.3em;
+		position: relative;
+
+		&:after {
```

Comment:
Not that it really matters much here, but if it's shown effectively before the cite, should we apply as `:before` for semantic's sake?